demos/common/samples/args_helper.hpp

```diff
@@ -26,6 +26,17 @@
 #include <dirent.h>
 #endif
 
+#define DEFINE_INPUT_FLAGS \
+static const char input_message[] = "Required. An input to process. The input must be a single image, a folder of " \
+    "images or anything that cv::VideoCapture can process."; \
+static const char loop_input_message[] = "Optional. Enable reading the input in a loop."; \
+DEFINE_string(i, "", input_message); \
+DEFINE_bool(loop_input, false, loop_input_message); \
+void showInputUsage() { \
+    std::cout << "    -i \"<path>\"               " << input_message << std::endl; \
+    std::cout << "    -loop_input               " << loop_input_message << std::endl; \
+}
```

Comment:
Mmm... it's a nice idea, but I think it's too fragile. You can't ensure that every demo will align its options with exactly the same amount of spaces. I think something like this _would_ be possible in the long run if we automated the formatting of help messages, but for now I we should just let every demo create the usage message by itself.

Reply:
Removed